mmv1/products/logging/LinkedDataset.yaml

```
@@ -125,10 +125,16 @@ examples:
   - !ruby/object:Provider::Terraform::Examples
     name: "logging_linked_dataset_basic"
     primary_resource_id: "logging_linked_dataset"
+    vars:
+      bucket_id: "my-bucket"
+      link_id: "mylink"
     test_env_vars:
       project: :PROJECT_NAME
   - !ruby/object:Provider::Terraform::Examples
     name: "logging_linked_dataset_all_params"
     primary_resource_id: "logging_linked_dataset"
+    vars:
+      bucket_id: "my-bucket"
+      link_id: "mylink"
```

```suggestion
      link_id: "my-link"
```

Could you add a dash here. Its a quirk/feature of our CI to prepend tf-test if a dash or underscore is present. Better to be consistent in the internally provisioned gear.

---

''